cmake/kokkos_arch.cmake

```diff
@@ -9,6 +9,52 @@ FUNCTION(KOKKOS_ARCH_OPTION SUFFIX DEV_TYPE DESCRIPTION)
   SET(KOKKOS_ARCH_${SUFFIX} ${KOKKOS_ARCH_${SUFFIX}} PARENT_SCOPE)
 ENDFUNCTION()
 
+FUNCTION(ARCH_FLAGS)
```

Comment:
Is this used at all?

Reply:
Is what used at all?

Reply:
I don't see that this function is used in this pull request. Where is it used/required?

Reply:
Did I mess something up in the cherry-pick?

Reply:
Here's the file diff of the PR to the release-candidate-3.o
https://github.com/kokkos/kokkos/pull/2705/files

Reply:
I add the link because I don't see the changes to add this function there, maybe it was removed in kokkos develop and my cherry-pick reintroduced it? I don't cherry-pick commits often and could have easily made the mistake.

Reply:
ooooh, Christian merged openmptarget stuff
Okay, I can fix this.

Reply:
For reference this is what I did:

```
cd <PATH_TO_KOKKOS>
git checkout release-candidate-3.0
git pull
git checkout develop
git pull
# create branch for this PR
git checkout -b cherry-pick-2705-from-rc
git cherry-pick 1f7128ab167f1205dbbc829fe8c0b59f6c6dc41b
# Then commit changes, push to my fork, and create PR
```

Reply:
@jjwilke if easier close this PR and start anew from the changes in #2705

Reply:
Yes, somehow the changes from 9bb9bd279 interfered.